src/lib/transition_frontier_controller_tests/test_transition_frontier.ml

```diff
@@ -110,8 +109,27 @@ let%test_module "Root_history and Transition_frontier" =
             ( breadcrumbs_path frontier query_hash
             |> Option.value_exn |> Non_empty_list.to_list ) )
 
+    let%test "moving the root removes the old root's non-heir children as \
+              garbage" =
+      Async.Thread_safe.block_on_async_exn (fun () ->
+          let%bind frontier = create_root_frontier ~logger in
+          let%bind () =
+            add_linear_breadcrumbs ~logger ~size:5 ~accounts_with_secret_keys
```

Comment:
is the 5 suppose to be a function of `max_length`? Can you define size as `max_length + 2`?

Reply:
this was supposed to be max_length actually.